frontend/src/pages/NewRunV2.tsx

```diff
@@ -214,6 +220,7 @@ function NewRunV2(props: NewRunV2Props) {
       description: runDescription,
       name: runName,
       pipeline_spec: {
+        pipeline_manifest: hasVersionID(apiRun) ? undefined : templateString,
```

Comment:
Do we also make sure that `resource_references` doesn't conflict with this field? So that there is always exclusive between `pipeline_manifest` and `resource_references` (line 231).